applications/serial_lte_modem/doc/GNSS_AT_commands.rst

```diff
@@ -69,6 +69,7 @@ Unsolicited notification
 
    #XGPS: <NMEA message>
 
+This is emitted when a fix is achieved and location sending to :ref:`nRF Cloud <SLM_AT_NRFCLOUD>` is enabled.
```

Comment:
I don't think we generally use the term "emitted" unless referring to light from LEDs. Is there another term that could be used?

Reply:
Would "transmitted" work as an alternative? I'm not saying "emitted" can't be used, it's just the first time I've seen it in this context in our documentation.